### PR TITLE
Move bash lint checks to GitHub actions.

### PR DESCRIPTION
**Pull request by paolino**
- [x]  moved todo list and bash linting to GH Coding Standard action

### CODE REVIEW

**Review by Anviking — APPROVED** [review]

File: .github/workflows/style.yml [hunk 1]
```diff
@@ -24,6 +24,7 @@ jobs:
     - name: '❄ Install Nix'
       uses: cachix/install-nix-action@v18
       with:
+        nix_path: nixpkgs=channel:nixos-unstable
```
Comment (Anviking) [inline .github/workflows/style.yml:27]: Or, why was this needed?

Comment (paolino) [reply .github/workflows/style.yml:27]: without that the script fail with 
```
warning: Nix search path entry '/nix/var/nix/profiles/per-user/kana/channels' does not exist, ignoring
error: file 'nixpkgs' was not found in the Nix search path (add it using $NIX_PATH or -I)

       at «string»:1:25:

            1| {...}@args: with import <nixpkgs> args; (pkgs.runCommandCC or pkgs.runCommand) "shell" { buildInputs = [ (nix-info) ]; } ""
             |                         ^
(use '--show-trace' to show detailed location information)

```

**Comment by paolino** [comment]
we stick with buildkite
